crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/meta_net_task.rs

```diff
@@ -413,9 +359,14 @@ impl MetaNetTask {
                                                                 .await
                                                             {
                                                                 Ok(op_hash) => op_hash,
-                                                                Err(_) => continue,
+                                                                Err(e) => {
+                                                                    tracing::warn!("Dropping incoming op because the host failed to hash it {:?}: {:?}", op, e);
+                                                                    continue
+                                                                },
                                                             };
 
+                                                            // TODO is this not too soon? We haven't actually stored the data yet
```

Comment:
Yeah, I can't think of any reason not to move this down below the `receive_ops` call.

Reply:
Great, I'll include that in my next PR. I haven't dug deep enough to be sure it's a benefit but likewise I can't see how it could be a negative